release.gradle

```diff
@@ -5,15 +5,15 @@ def isReleaseBuild() {
     return VERSION_NAME.contains("SNAPSHOT") == false
 }
 
-def getRepositoryUrl() {
+def getMavenRepositoryUrl() {
```

Comment:
This change is necessary with a more recent version of Gradle and/or the Android Gradle plugin. It seems like there is a naming clash with the property `repositoryUrl` the auto-generated getter method.

Without this change the build will fail with this error message:

```
FAILURE: Build failed with an exception.

* Where:
Script '[path]/release.gradle' line: 9

* What went wrong:
A problem occurred configuring project ':stetho'.
> Could not get unknown property 'repositoryUrl' for project ':stetho' of type org.gradle.api.Project.
```